### PR TITLE
feat: add OpenGraph meta tags

### PR DESCRIPTION
**Pull request by sarscodes**
- [x] I've checked that this isn't a duplicate pull request.

Added [OpenGraph](https://ogp.me) and [Twitter Card](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup) Meta tags to enhance share preview on social media.

### CODE REVIEW

File: src/pug/includes/head.pug [hunk 1]
```diff
@@ -7,6 +7,14 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
 
 meta(name="description" content="swag opportunities for developers")
 
+meta(property="og:title" content="devSwag")
+meta(property="og:description" content="Swag opportunities for developers")
```
Comment (vikaspotluri123) [inline src/pug/includes/head.pug]:
```suggestion
meta(property="og:description" content="swag opportunities for developers")
```

We either need to update the meta description, or keep this consistent 😄 ping @aslafy-z

File: src/pug/includes/head.pug [hunk 2]
```diff
@@ -7,6 +7,14 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
...
+meta(property="og:site_name" content="devSwag")
+meta(property="og:type" content="website")
+meta(property="og:url" content="https://devswag.io")
+meta(property="og:image" content="/assets/img/devswag-card.png")
```
Comment (vikaspotluri123) [inline src/pug/includes/head.pug]: Did you make this?

**Comment by vikaspotluri123** [comment]
Do we want to add JSON-LD metadata as well?

File: src/pug/includes/head.pug (hunk 2 above)
Comment (sarscode) [reply src/pug/includes/head.pug]: Yes, I did this to create a preview photo on cards

Comment (vikaspotluri123) [reply src/pug/includes/head.pug]: Alright, just checking! I'll let the other maintainers review it, but I'd personally  prefer a bit more whitespace after the divider

File: src/pug/includes/head.pug (hunk 1 above)
Comment (aslafy-z) [reply src/pug/includes/head.pug]: I prefer the lowercase one, @swapagarwal? 💯

Comment (sarscode) [reply src/pug/includes/head.pug]: It's nice to have the `description` consistent. I have committed an update to use a lowercase `s` @vikaspotluri123 and @aslafy-z  for the suggestion.

File: src/pug/includes/head.pug (hunk 2 above)
Comment (sarscode) [reply src/pug/includes/head.pug]: @aslafy-z and @swapagarwal what do you think about the photo card

Comment (aslafy-z) [reply src/pug/includes/head.pug]: It looks good! But I'm okay with @vikaspotluri123, I think it needs some more spacing.

**Comment by aslafy-z** [comment]
I think we should rename the card to `card.png` or `social-card.png`

**Comment by aslafy-z** [comment]
@vikaspotluri123 let's do it in another PR

**Comment by sarscode** [comment]
@vikaspotluri123 and @swapagarwal could please review this. Let's have your take on this, let me know if I have to make another pull request.
Thanks

File: src/pug/includes/head.pug (hunk 2 above)
Comment (vikaspotluri123) [reply src/pug/includes/head.pug]: Just waiting on this 😄

Comment (vikaspotluri123) [reply src/pug/includes/head.pug]: ping @sarscode

**Comment by aslafy-z** [comment]
@sarscode can you have a look at https://github.com/swapagarwal/swag-for-dev/pull/351#discussion_r334503433?

File: src/pug/includes/head.pug (hunk 2 above)
Comment (aslafy-z) [reply src/pug/includes/head.pug:15]: ping @sarscodes, any chance you get some time to edit this photo card? It would be awesome!

**Comment by aslafy-z** [comment]
This PR will be replaced by https://github.com/swapagarwal/swag-for-dev/pull/871 since we have no news from you @sarscodes.
Thank you for the effort.

**Pull request closed without merging**